frontend/src/services/user.ts

```diff
@@ -0,0 +1,25 @@
+import {AuthCredentials} from "../models/authCredentials";
+import {ILoggedUser} from "../types/loggedUser";
+import {UserEntity} from "../models/userEntity";
+import {UserType} from "../types/UserType";
+
+export const logInService = async (authCredentials: AuthCredentials): Promise<string> => {
+   return "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJjdXN0b21lcjFAZ21haWwuY29tIiwiaWF0IjoxNzExOTYyMDM4fQ.g5oDNUFe6MwsfP7syojOKMTd75OXiSL0VGCPEc2aoHg";
+}
+
+export const getUser = async (token: string): Promise<ILoggedUser> => {
+    const userEntity: UserEntity = {
+        id: 0,
+        email: "user@gmail.com",
+        name: "User",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+    }
+    return {
+        id: userEntity.id,
+        email: userEntity.email,
+        username: userEntity.name,
+        userType: UserType.Customer,
+        token: token
```

Comment:
It's not a task of user service to implicitly set token. it has to return user, that's all